components/kronos/header/DropdownAccnt.vue

```diff
@@ -2,15 +2,15 @@
   <CDropdown in-nav class="c-header-nav-items" placement="bottom-end" add-menu-classes="pt-0">
     <template #toggler>
       <CHeaderNavLink>
-        <Avatar :username="$me.name || 'Anon'" />
+        <Avatar :username="$ssoScbd.me.name" />
       </CHeaderNavLink>
     </template>
     <CDropdownHeader tag="div" class="text-center" color="light">
       <strong>Account</strong>
     </CDropdownHeader>
     <CDropdownDivider />
     <CDropdownItem
-      :href="`${$auth.accountsUrl}/profile/?returnIUrl=${returnUrl}`"
+      :href="`${$ssoScbd.auth.accountsUrl}/profile/?returnIUrl=${returnUrl}`"
```

Comment:
`returnUrl` needs encodeURIComponent
```suggestion
      :href="`${$ssoScbd.auth.accountsUrl}/profile/?returnIUrl=${encodeURIComponent(returnUrl)}`"
```

Reply:
not sure encodeURIComponent is accessible in the context

Reply:
I think it will work in this context.  I read somewhere to try and reduce js inside the interpolation. returnUrl is a computed prop/function which encodes already. https://github.com/scbd/kronos-web/blob/ba588b9d1fe7337b7b987c348b3254bb3703537b/components/kronos/header/DropdownAccnt.vue#L40

Reply:
the qs parameter id mispelled `returnIUrl` instead of `returnIUrl`

Reply:
https://github.com/scbd/kronos-web/blob/ba588b9d1fe7337b7b987c348b3254bb3703537b/components/kronos/header/DropdownAccnt.vue#L40
the url retuneed by the returnUrl prop should no be encoded other it's not an URL.

returnUrl prop should return a valid Url
when you pass a value into a query string then you have to encode it and only at that time.

Reply:
The target is _blank.  It should not have any returnUrl I think?

Reply:
@sbilodeau this is outdated.  I pushed a resolve.